src/webnn_native/dml/GraphDML.cpp

```diff
@@ -1092,6 +1092,31 @@ namespace webnn_native { namespace dml {
         return {};
     }
 
+    MaybeError Graph::AddSplit(const op::Split* split) {
+        DAWN_ASSERT(split->Inputs().size() == 1);
+        const OperandBase* inputOperand = split->Inputs()[0].Get();
+        DAWN_ASSERT(mExpression.find(inputOperand) != mExpression.end());
+        ::dml::Expression input = mExpression.at(inputOperand);
+        ::dml::TensorDimensions inputDims = input.GetOutputDesc().sizes;
+
+        // dml::Span just holds the refernces, need a variable to hold the memory.
+        std::vector<uint32_t> splits = split->GetSplits();
+        int32_t axis = split->GetAxis();
+        if (splits.size() == 1) {
+            if (inputDims[axis] % splits[0] != 0) {
+                return DAWN_INTERNAL_ERROR("The splits can't be splitted.");
```

Comment:
I would suggest the error message could be specific, such as "the axis x with size n can't be divisible by splits[0] m" where x, n, m are actual values.